apps/saleor-app-checkout/.env

```diff
@@ -22,3 +20,7 @@ NEXT_PUBLIC_SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}
 NEXT_PUBLIC_GIT_BRANCH=${VERCEL_GIT_COMMIT_REF}
 SENTRY_RELEASE=${VERCEL_GIT_COMMIT_SHA}
 NEXT_PUBLIC_SENTRY_RELEASE=${VERCEL_GIT_COMMIT_SHA}
+
+APL=file
+UPSTASH_REDIS_REST_URL=
+UPSTASH_REDIS_REST_TOKEN=
```

Comment:
These envs are not required, are confusing - because `APL=file` is exclusive to them.

Every "APL" will have its own, unique set of env variables to be configured with. I'd remove them and add comment that link to docs https://github.com/saleor/saleor-app-sdk/blob/main/docs/apl.md